mkosi/__init__.py

```diff
@@ -2304,6 +2304,10 @@ def install_fedora(args: CommandLineArguments, root: str, do_run_build_script: b
     with open(os.path.join(root, "etc/locale.conf"), "w") as f:
         f.write("LANG=C.UTF-8\n")
 
+    # FIXME: should this be conditionalized on args.with_docs like in install_debian_or_ubuntu()?
+    #        But we set LANG=C.UTF-8 anyway.
+    shutil.rmtree(os.path.join(root, "usr/share/locale"))
```

Comment:
isn't there an official way to get rid of this in fedora?

Reply:
Not directly. `rpm` has `--excludepaths`, but I'm not sure if there's a way to tell dnf to pass that.

Reply:
https://bugzilla.redhat.com/show_bug.cgi?id=1967970

Reply:
hmm, maybe we should add a new generic RemoveFiles= stanza that works on all distros, and can be used here? It would just list paths (with wildcards) to remove as very last step of the build or so?

Reply:
And then maybe explicit provide an example in the docs, that it's a good idea to list /usr/share/locale in this setting

Reply:
`RemoveFiles=` sounds good. But let's do that separately, this one is already getting rather complex.